client/post-editor/post-editor.jsx

```diff
@@ -713,9 +713,6 @@ export const PostEditor = React.createClass( {
 			this.props.resetEditorLastDraft();
 		}
 
-		// Remove this when the editor is completely reduxified ( When using Redux actions for all post saving requests )
-		this.props.savePostSuccess( post.site_ID, this.props.postId, post, {} );
```

Comment:
In my testing I haven't found any issues with removing this code.  The same logic is repeated by [exitPost](https://github.com/Automattic/wp-calypso/blob/master/client/post-editor/controller.js#L233) which seems to clear up the original issue of lingering draft edits when leaving the editor.

> So maybe the fix for this would be to clear the edits when triggering the request instead of when receiving the response.

This seems like a logical approach too.  The action would have to clear edits specific for autoSave vs full save events since autoSave only sends Title, Excerpt and content so it should not clear out edits of other attributes.

Would you be willing to allow me to explore that approach in a subsequent PR @youknowriad and at least get this fix out to stop the loss of data?  I hit this bug a few times yesterday while testing the new chrome and it really does take ones trust away from the save operations.